src/go/k8s/cmd/configurator/Dockerfile

```diff
@@ -15,10 +15,10 @@ COPY k8s/go.sum go.sum
 RUN go mod download
 
 # Copy the go source
-COPY k8s/cmd/configurator/main.go main.go
+COPY k8s/ .
```

Comment:
nit: why this change?

Reply:
Because of https://github.com/vectorizedio/redpanda/pull/782/files#diff-d8d3d68da6374674c5a5fae39e3220fdfa6a581caa5ef01f675aa3757f9a9d80R107